nucypher/cli/deploy.py

```diff
@@ -116,12 +116,13 @@ def inspect(provider_uri, config_root, registry_infile, deployer_address, poa):
 @click.option('--retarget', '-d', help="Retarget a contract's proxy.", is_flag=True)
 @click.option('--target-address', help="Recipient's checksum address for token or ownership transference.",
               type=EIP55_CHECKSUM_ADDRESS)
+@click.option('--ignore-deployed', help="Ignore already deployed contracts if exist.", is_flag=True)
```

Comment:
return of "bare" mode? I like your name better.

Reply:
It's related to "bare" but much simpler, turns off 'is_deployed' check to redeploy same contract multiple times (first of all for test)

Reply:
Just for my own understanding: is this flag essentially a `force` type option to deploy the contract even if previously deployed?

Reply:
Correct, one remark - `force` for the same contract with the same version

Reply:
👍

Reply:
Does this have a use beyond tests?

Reply:
I see at least one use: redeploying same contract but with different constructor parameters, for example, disable/enable test mode of StakingEscrow (for worklock)

Reply:
But are we going to do that on a real deployment? because as far as I understand, the test mode is only for tests. Or are you planning on trying that on testnet?

Reply:
test mode for testnet, to change worklock